### PR TITLE
Enable repos when install medium are dvds

### PR DESCRIPTION
**Pull request by foursixnine**
https://progress.opensuse.org/issues/48134

### CODE REVIEW

File: tests/console/zypper_ref.pm [hunk 1]
```diff
@@ -18,6 +19,12 @@ use utils 'zypper_call';
 
 sub run {
     select_console 'root-console';
+
+    # see FATE#325541
+    zypper_call 'mr -e -l'
```
Comment (rwx788) [inline tests/console/zypper_ref.pm]: That's not the third place we got it =(. Would you be the one who can put single implementation?
`tests/autoyast/repos.pm`
`tests/sles4sap/patterns.pm` 
are the other 2 places...

Comment (foursixnine) [reply tests/console/zypper_ref.pm]: @rwx788 indeed, thought about creating a follow-up ticket to do the change.

Comment (rwx788) [reply tests/console/zypper_ref.pm]: One way or another, feel free to merge this PR.

Comment (foursixnine) [reply tests/console/zypper_ref.pm]: I guess I'll merge for now, pick the followup later in the sprint

Comment (rwx788) [reply tests/console/zypper_ref.pm]: Sure, let's refactor later on

**Comment by nilxam** [comment]
Test died: Undefined subroutine &zypper_ref::is_sle called at /var/lib/openqa/cache/openqa1-opensuse/tests/opensuse/tests/console/zypper_ref.pm line 24.

**Comment by rwx788** [comment]
Ouch. @foursixnine will you provide patch.

**Comment by foursixnine** [comment]
Yep

**Pull request merged**